src/components/Navbar/Toolbar.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBars, faTimes } from '@fortawesome/free-solid-svg-icons';
+import { Link, useLocation } from 'react-router-dom';
+import PropTypes from 'prop-types';
+const Toolbar = ({ navbarOpen, handleToggleMenuClick }) => {
+  const location = useLocation();
+  const logoColor =
```

Comment:
we can put this condition in one variable to save ourselves from comparing 3 times
```suggestion
  const isDark = location.pathname === '/home' || location.pathname === '/about';
```

and then using the `isDark` to check,

I would also highly recommend checking this library 
https://www.npmjs.com/package/classnames

it does couple of good stuff and one of them is conditional css classes
you can read more about it here
https://www.pluralsight.com/guides/applying-classes-conditionally-react

so if you used this the right way you should be able to remove all these lines and only keep the `isDark`

Reply:
That was helpful dude !! Thank you !!